### ISSUE TITLE
Deleted FormData

### ISSUE THREAD

**Issue by maxdenaro** [horprogs/Just-validate#45]
Hi! I tried to send my form with just-validate (new syntax, 3.0.0 and more), but formData just cleared. 

On older versions where the attributes are still present "data-validate-field" this does not happen.

```
.onSuccess((event) => {
      const form = document.querySelector('.form');
      let formData = new FormData(form); // this formData is empty on new versions JustValidate
```

**Comment by horprogs** [comment]
Hi! Are you sure you use the correct form selector? Also it makes sense to prevent default submitting form, I guess it could be a reason of your issue:

```
.onSuccess((ev) => {
  ev.preventDefault();
  const form = document.querySelector("#basic-validation-form");
  let formData = new FormData(form);
  console.log(formData);
});
```

Check it out in this example: https://codesandbox.io/s/elastic-lalande-2bxjj?file=/index.html

**Comment by maxdenaro** [comment]
Hi! Yeah, selector is correct, but it doesnt work.

```
.onSuccess((ev) => {
    ev.preventDefault();
    const form1 = document.querySelector(".form");
    let formData = new FormData(form1);
    console.log(formData);
    console.log('Validation passes and form submitted', ev);

    console.log(...formData);

    for (var pair of formData.entries()) {
        console.log(pair[0]+ ', ' + pair[1]);
    }
  });
```

in the old version, with this output, или если использовать обычное событие submit, everything is displayed in the console, and ends up in $_POST in PHP, but in the new version of Just Validate formData is empty.

formData filled only if you manually use the method append().

**Comment by maxdenaro** [comment]
my working code on old version - https://github.com/maxdenaro/maxgraph-youtube-source/tree/master/UI-%D0%BA%D0%BE%D0%BC%D0%BF%D0%BE%D0%BD%D0%B5%D0%BD%D1%82%D1%8B%20%E2%84%9613.%20%D0%92%D0%B0%D0%BB%D0%B8%D0%B4%D0%B0%D1%86%D0%B8%D1%8F%20%D0%B8%20%D0%BE%D1%82%D0%BF%D1%80%D0%B0%D0%B2%D0%BA%D0%B0%20%D1%84%D0%BE%D1%80%D0%BC%D1%8B%20%D0%BD%D0%B0%20%D0%BF%D0%BE%D1%87%D1%82%D1%83%20%D0%B2%202022

**Comment by horprogs** [comment]
okay, I see, let me check

**Comment by horprogs** [comment]
Thanks for the great catch, was fixed in `3.3.2`
